public/app/core/components/Breadcrumbs/utils.ts

```diff
@@ -35,7 +35,16 @@ export function buildBreadcrumbs(sectionNav: NavModelItem, pageNav?: NavModelIte
     const shouldAddCrumb = !node.hideFromBreadcrumbs && !(shouldDedupe && isSamePathAsLastBreadcrumb);
 
     if (shouldAddCrumb) {
-      crumbs.unshift({ text: node.text, href: node.url ?? '' });
+      const activeChildIndex = node.children?.findIndex((child) => child.active) ?? -1;
```

Comment:
Doesn't `findIndex` return `-1` by default if no elements found?

Reply:
yes, but `node.children` could be `undefined`